saturn-datepicker/src/datepicker/month-view.html

```diff
@@ -10,11 +10,12 @@
          [selectedValue]="_selectedDate"
          [begin]="_beginDateNumber"
          [end]="_endDateNumber"
+         [beginSelected]="!!_beginDateSelected"
+         [isBeforeSelected]="_beginDateSelected && _dateAdapter.compareDate(activeDate, _beginDateSelected) < 0"
          [rangeFull]="_rangeFull"
          [rangeMode]="rangeMode"
          [labelMinRequiredCells]="3"
          [activeCell]="_dateAdapter.getDate(activeDate) - 1"
-         (selectedValueChange)="_dateSelected($event)"
-         (keydown)="_handleCalendarBodyKeydown($event)">
+         (selectedValueChange)="_dateSelected($event)">
```

Comment:
and don't forget to return this back

Reply:
https://github.com/SaturnTeam/saturn-datepicker/pull/57/commits/f618d69c5275f26a68291f1410b743bb5bec5264